drivers: drm: panel: Add ILITEK ili7807-fdhplus panel driver

qwertyuiopzxcvbnmlkjhgfdsa1 opened the pull request:
Adding generated driver for panel ili8707-fhdplus
This adds support for ILITEK ili7807-fdhplus 1080x2280 video
mode panel that can be found on Xiaomi A2 Lite(daisy) device.


Signed-off-by: Petr Blaha <petr.blaha@cleverdata.cz>

SirSireesh commented:
Hi, thanks for making this commit! Could you please reword the commit message? And squash the commits? Here's an example of a good commit message: https://github.com/msm8916-mainline/linux/commit/02ed76b6da0710a9d24f36a4a9800a68d7edb9a7

qwertyuiopzxcvbnmlkjhgfdsa1 commented:
> Hi, thanks for making this commit! Could you please reword the commit message? And squash the commits? Here's an example of a good commit message: [msm8916-mainline@02ed76b](https://github.com/msm8916-mainline/linux/commit/02ed76b6da0710a9d24f36a4a9800a68d7edb9a7)

Renamed commit message and squoshed. Added sign off by, thanks for hint

qwertyuiopzxcvbnmlkjhgfdsa1 commented:
> The text looks ok, but could you please add it to the commit itself?

@SirSireesh  Text added to commit. Is it ok?

SirSireesh commented:
Yes, looks fine now. What about the device tree changes?

qwertyuiopzxcvbnmlkjhgfdsa1 commented:
> I think you removed the code that enables the regulators. If you are not using mipi-dsi-common you need to generate code for every regulator on the DTS, in this case vsp and vsn.

Added the regulators, thanks for hint.

SirSireesh commented:
Eep, I was asking for device tree changes because I didn't realize the `compatible` string adjustments was being handled by lk2nd for daisy. These changes are good to go now, merging!
